bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -597,6 +601,18 @@ external-dns: transip.apiKey
 {{- end -}}
 {{- end -}}
 
+{{/*
+Validate values of External DNS:
+- must provide the Linode API token when provider is "linode"
+*/}}
+{{- define "external-dns.validateValues.linode.apiToken" -}}
```

Comment:
I think the call to this function is missing.

Reply:
What do you expect to see, @miguelaeh?

I've added https://github.com/bitnami/charts/pull/4080/files#diff-558186ea703aa211d25e6d42025193951967788ee11f1bbddeedcce7d2393dc6R258, but again, not all providers include these lines (Google, DO, Hetzner), so I'm not sure if I should be including it.

It may be beneficial to compare this PR against the recently merged Hetzner PR, https://github.com/bitnami/charts/pull/4424/files.

Reply:
The tests are not passing on this PR, but they do pass on the Hetzner PR.

```
>>> helm install external-dns-xvijokkovr bitnami/external-dns --namespace external-dns-xvijokkovr --wait
Error: parse error at (external-dns/templates/deployment.yaml:616): unexpected EOF
```

hmm

Reply:
Tests are passing now - I was missing some `{{- end }}` from a bad resolution on the merge conflicts.

Reply:
Sorry for this I missed it, the call to the function is actually correct.